src/fidesops/migrations/versions/55d61eb8ed12_add_default_policies.py

```diff
@@ -0,0 +1,384 @@
+FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
+AUTOGENERATED_ACCESS_KEY = "download"
+AUTOGENERATED_ERASURE_KEY = "delete"
+
+client_select_query: TextClause = text(
+    """SELECT client.id FROM client WHERE fides_key = :fides_key"""
+)
+
+client_insert_query: TextClause = text(
+    """
+    INSERT INTO client (id, hashed_secret, salt, scopes, fides_key) 
+    VALUES (:client_id, :hashed_secret, :salt, :scopes, :fides_key)
+    """
+)
+
+client_delete_query: TextClause = text(
+    """DELETE FROM client WHERE client.id = :client_id"""
+)
+
+storage_select_query: TextClause = text(
+    """SELECT storageconfig.id FROM storageconfig WHERE storageconfig.key = :storage_config_key"""
+)
+storage_insert_query: TextClause = text(
+    """
+    INSERT INTO storageconfig(id, name, type, details, key, secrets, format) 
+    VALUES(:storage_config_id, :storage_name, :storage_type, '{"naming": "request_id"}', 
+    :storage_key, NULL, 'json')
+    """
+)
+storage_delete_query: TextClause = text(
+    """DELETE FROM storageconfig WHERE storageconfig.id = :storage_id"""
+)
+policy_select_query: TextClause = text(
+    """
+    SELECT policy.id FROM policy WHERE policy.key = :policy_key AND client_id = :client_id
+    """
+)
+policy_insert_query: TextClause = text(
+    """
+    INSERT INTO policy(id, name, key, drp_action, client_id) 
+    VALUES(:policy_id, :policy_name, :key, :drp_action, :client_id)
+    """
+)
+rule_select_query = text("""SELECT rule.id FROM rule where policy_id = :policy_id""")
+rule_insert_query: TextClause = text(
+    """
+    INSERT INTO rule (id, name, key, policy_id, action_type, masking_strategy, storage_destination_id, client_id) 
+    VALUES (:rule_id, :rule_name, :rule_key, :policy_id, :action_type, :masking_strategy, :storage_id, :client_id)
+    """
+)
+rule_target_insert_query: TextClause = text(
+    """
+    INSERT INTO ruletarget (id, name, key, data_category, rule_id, client_id) 
+    VALUES (:target_id, :target_name, :target_key, :data_category, :rule_id, :client_id)
+    """
+)
+
+delete_policy_query = text("""DELETE FROM policy WHERE policy.id = :policy_id""")
+delete_rule_query = text("""DELETE FROM rule WHERE policy_id = :policy_id""")
+delete_target_query = text("""DELETE FROM ruletarget WHERE rule_id IN :rule_ids""")
+
+
+def generate_uuid(cls: FidesopsBase) -> str:
+    """
+    Generates a uuid with a prefix based on the tablename to be used as the
+    record's ID value
+    """
+    try:
+        prefix = f"{cls.__tablename__[:3]}_"
+    except AttributeError:
+        prefix = ""
+    uuid = str(uuid4())
+    return f"{prefix}{uuid}"
+
+
+def upgrade() -> None:
+    """Data migration only.
+
+    Create an autogenerated client and storage destination, then use those to create
+    autogenerated 'download' and 'delete' policies if they don't already exist."""
+    if config.is_test_mode:
+        logger.info(f"Skipping data migration in test mode'")
+        return
```

Comment:
It seemed like we shouldn't be adding these to the test database, so I run the migration, but don't insert data.

Reply:
This is an interesting pattern to set. I could see us wanting to test these policies specifically at some stage, we can address this then.